packages/components/navigation/BuyTokens.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Linking, TouchableOpacity } from "react-native";
+import { CreditCardIcon } from "react-native-heroicons/outline";
+
+import { primaryColor } from "../../utils/style/colors";
+import { fontBold16 } from "../../utils/style/fonts";
+import { BrandText } from "../BrandText";
+
+export const BuyTokens: React.FC = () => {
+  return (
+    <TouchableOpacity
+      style={{
+        justifyContent: "center",
+        alignItems: "center",
+        marginBottom: "0.5em",
+        flexDirection: "row",
+      }}
+      onPress={() => {
+        Linking.openURL("https://frontier.osmosis.zone/?from=OSMO&to=TORI");
+      }}
+    >
+      <CreditCardIcon color={primaryColor} style={{ marginRight: "0.5em" }} />
```

Comment:
You should use the same space value as the other elements in the Sidebar.
Like padding.layout_x1

Reply:
no `em` see https://reactnative.dev/docs/height-and-width